frontend/src/components/store/PluginCard.tsx

```diff
@@ -137,77 +128,82 @@ const PluginCard: FC<PluginCardProps> = ({ storePlugin, installedPlugin }) => {
             </div>
           )}
         </div>
-        <div className="deckyStoreCardButtonRow">
-          <PanelSectionRow>
-            <Focusable style={{ display: 'flex', gap: '5px', padding: 0 }}>
-              <div
-                className="deckyStoreCardInstallContainer"
-                style={
-                  {
-                    paddingTop: '0px',
-                    paddingBottom: '0px',
-                    flexGrow: 1,
-                    '--field-negative-horizontal-margin': 0,
-                  } as CSSProperties
-                }
+        <PanelSectionRow>
```

Comment:
Sorry for the weird diff here (GitHub diff is not great) 🫤 This is just an unneeded `<div>` removed but the nesting removal is not shown very clearly here on GH. This is more clear in e.g. VSCode diff view.